pkg/kubeadm/master/addons.go

```diff
@@ -28,11 +29,14 @@ import (
 	ipallocator "k8s.io/kubernetes/pkg/registry/service/ipallocator"
 	"k8s.io/kubernetes/pkg/util/intstr"
 )
-
-func createKubeProxyPodSpec(s *kubeadmapi.KubeadmConfig) api.PodSpec {
+// TODO(phase1+): kube-proxy should be a daemonset, three different daemonsets should not be here
```

Comment:
Yes but why do it for multiple (in this case, three) architectures and not just the one where `kubeadm init` ran?